feconf.py

```diff
@@ -778,3 +778,7 @@ def get_empty_ratings():
     'distributing learning material.')
 THANKS_PAGE_DESCRIPTION = (
     'Thank you for donating to The Oppia Foundation.')
+
+# The type returned by a handler when exception raised
```

Comment:
The type of the response returned by a handler when an exception is raised.

(don't forget to end with a full stop)